### PR TITLE
Feat/user page

### PR DESCRIPTION
**Pull request by niksonax**

### CODE REVIEW

File: frontend/src/components/Main/index.tsx [hunk 1]
```diff
@@ -24,10 +25,10 @@ function Main() {
       <PopoverExample />
       <ModalWindow
         show={show}
-        title="Modal"
-        body="Hello there!"
-        size="medium"
-        centered
+        title="User Page"
+        body={<UserPage />}
+        size="user-page"
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Main/index.tsx]: It's a shared component, so rename it, eg extra-wide.

File: frontend/src/scenes/UserPage/components/Body.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  projects?: Project[];
+  achievements?: Achievement[];
+}
+
+const CardsGrid: FunctionComponent<CardsGridProps> = ({
+  target,
+  projects = [],
+  achievements = []
+}) => {
+  if (target === 'projects') {
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/UserPage/components/Body.tsx]: Let's use component Tab here

File: frontend/src/scenes/UserPage/components/Body.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    const renderProjectsList = () => projects.map((project) => (
+      <ProjectCard
+        key={project.id}
+        to={project.url}
+        name={project.name}
+        description={project.description}
+        category={project.category}
+        tags={project.tags}
+        image={project.imageUrl}
+        percent={Math.floor((project.donated / project.goal) * 100)}
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/UserPage/components/Body.tsx]: This function might be used in multiple places, move it to helpers

File: frontend/src/scenes/UserPage/components/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+              <FontAwesomeIcon icon={faDribbble} size="2x" />
+            </a>
+            )}
+        </div>
+      </div>
+
+      <div className={classes['user-about-me']}>
+        <h3>About Me</h3>
+        <p>
+          {aboutMe} {/* Make 240 symbols limit */}
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/UserPage/components/Header.tsx]: comment

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]
Is it optimised for mobile phones?

**Comment by niksonax** [comment]
> Is it optimised for mobile phones?



> Is it optimised for mobile phones?
No, it's not. But I can work on it today

**Comment by niksonax** [comment]
> Is it optimised for mobile phones?

No, it`s not. I can work on it today to make it adaptive

**Pull request merged**